images/versioned/cuda.md

```diff
@@ -31,13 +31,8 @@ title: cuda, ml, ml-verse
 These images correspond to [`rocker/r-ver`](r-ver.md),
 [`rocker/tidyverse`, and `rocker/geospatial`](rstudio.md), respectively.
 
-All images are based on the current Ubuntu LTS and based on the official [NVIDIA CUDA docker build recipes](https://gitlab.com/nvidia/container-images/cuda/).
-
-:::{.callout-note}
-
-Older images, `rocker/ml-gpu`, `rocker/tensorflow` and `rocker/tensorflow-gpu`, built with cuda 9.0, are deprecated and no longer supported.
-
-:::
+All images are based on the official [NVIDIA CUDA docker build recipes](https://gitlab.com/nvidia/container-images/cuda/),
```

Comment:
This is true since "based on" is a bit loose terminology, but not sure if it is worth noting that the 10.1 cuda tags aren't technically built via a `FROM:` directive on the official nvidia tagged images, but rather just runs much the same scripts on top of the standard rocker script.  with 11.1 and going forward we literally build `FROM:` the nvidia images.

Reply:
Is there any reason to install cuda 10.1 on `rocker/r-ver` without changing the base image?
Slightly off topic here, but it does seem a bit odd that the cuda11 image and the cuda10 image are built on different layers rather than a different base image.

I also don't know why we continue to use the old cuda 10.1 instead of keeping only the latest cuda version.

Reply:
@eitsupi oh sorry -- NVIDIA doesn't have a 10.1 CUDA that is based on ubuntu20.04; only ubuntu18.04. 

yeah I don't think we want to continue with old cuda 10.1 on new images.  It makes more sense that our ubuntu20.04/focal images go with 11.1 I think; 10.1 need only be available in the older locked versions.  I think when we bump to 22.04 base image, the ML stack should bump to the 22.04-base cuda 11.7.0 NVIDIA base image.  Does that sound reasonable?

Reply:
Ok, here is a new issue about that. rocker-org/rocker-versioned2#503.